src/main/java/shadows/apotheosis/mixin/MixinBambooBlock.java

```diff
@@ -0,0 +1,36 @@
+package shadows.apotheosis.mixin;
+
+import net.minecraft.block.BambooBlock;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.Constant;
+import org.spongepowered.asm.mixin.injection.ModifyConstant;
+
+import shadows.apotheosis.garden.GardenModule;
+
+@Mixin(BambooBlock.class)
+public class MixinBambooBlock {
+	@ModifyConstant(method = {
+		"randomTick",
+		"canGrow",
+		"grow(Lnet/minecraft/world/server/ServerWorld;Ljava/util/Random;Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/block/BlockState;)V",
+		"getNumBambooBlocksAbove",
+		"getNumBambooBlocksBelow"
+	}, constant = @Constant(intValue = 16))
+	public int modifyMaxBambooHeight(int maxHeightIn) {
```

Comment:
Mixin injectors should be **`private`**. `public` injectors cannot be obfuscated at runtime and run a significantly higher risk of conflict.